x-pack/plugins/alerting/docs/openapi/components/schemas/notify_when.yaml

```diff
@@ -1,12 +1,13 @@
 type: string
 description: >
+  Deprecated in 8.7.0. Use the `notify_when` property in the action `frequency` object instead.
```

Comment:
>  Deprecated in 8.7.0. Use the `notify_when` property in the action `frequency` object instead.

If we're going to declare this deprecation as of 8.13.1, this will need to be updated. Otherwise, I'll need to go back and update the 8.7.0 release notes.